### PR TITLE
Activity data as % of total population

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for #660 

Integration Donut Chart

Before : 
Screenshot 2023-07-26 at 2 12 52 PM

After:
Screenshot 2023-07-26 at 2 13 19 PM

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 20 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 20 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/src/containers/MainPage/Metrics/ActivityDonutChart.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+Chart.register(CategoryScale);
+Chart.register(ChartDataLabels);
+
+const FETCH_ACTIVITY_PERCENTAGE_URL = '/activity-percentage/'
+
+const ActivityDonutChart = () => {
+    const [loading, setLoading] = useState(false)
+    const [activityData, setActivityData] = useState([])
+    const activityType = {
+        "Unplanned/illegal hunting": "#FF5252",
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Metrics/ActivityDonutChart.tsx:21]: I understand that they wanted this to be static, but let's implement that in the backend instead. This way, we won't need to write the activity type twice.

Comment (ketankartoza) [reply django_project/frontend/src/containers/MainPage/Metrics/ActivityDonutChart.tsx:21]: @dimasciput Should I send all the activity_type with activity_type's colour from the back-end?

Comment (ketankartoza) [reply django_project/frontend/src/containers/MainPage/Metrics/ActivityDonutChart.tsx:21]: @dimasciput Should I send all the activity_type with activity_type's colour from the back-end?

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 11 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 19 min

**Pull request merged**